en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -37,20 +37,11 @@ We officially support and regression test against the latest LTS releases of MyS
 ### Default MySQL collation
 
 New projects default to the `utf8mb4_unicode_ci` collation when running
-against MySQL, which offers better support for multi-byte characters such as emoji. However, this may cause issues
-related to Varchar fields exceeding the maximum indexable size:
+against MySQL, which offers better support for multi-byte characters such as emoji.
 
-- MySQL 5.6 supports larger indexes (3072 bytes) if the `innodb_large_prefix` setting is enabled (but not by default)
-- MySQL 5.7 and newer have `innodb_large_prefix` enabled by default
-- MariaDB ~10.1 matches MySQL 5.6's behaviour, >10.2 matches 5.7's.
+### Connection mode (sql_mode) when using MySQL server >=8.0.0
 
-You can rectify this issue by upgrading MySQL, enabling the `innodb_large_prefix` setting if available, or reducing the
-size of affected fields. If none of these solutions are currently suitable, you can remove the collation
-configuration from `app/_config/mysite.yml` to default back to the legacy default collation.
-
-### Connection mode (sql_mode) when using MySQL server >=5.7.5
-
-In MySQL versions >=5.7.5, the `ANSI` sql_mode setting behaves differently and includes the `ONLY_FULL_GROUP_BY`
+In MySQL versions >=8.0.0, the `ANSI` sql_mode setting behaves differently and includes the `ONLY_FULL_GROUP_BY`
```

Comment:
Do we know what this means by "behaves differently"? I assume it means "behaves differently to older versions". If that's the case, we should remove that part so it just reads
> In MySQL versions >=8.0.0, the `ANSI` sql_mode setting  includes the ...

Reply:
Updated